src/components/services-view/services-view.controller.ts

```diff
@@ -131,4 +183,20 @@ export class ServicesViewController implements angular.IController {
   private updateActiveCardStyles() {
     this.$timeout(this.resizeExpansion, 50);
   }
-}
+
+  private updateFilterControls() {
+    this.$timeout(() => {
+      if (this.ctrl.filterConfig.appliedFilters.length > 0) {
+        this.ctrl.filterConfig.resultsCount = this.ctrl.filteredItems.length;
+        $('.toolbar-pf-results h5').text(this.ctrl.filterConfig.resultsCount + ' of ' + this.ctrl.totalCount + ' items');
+      } else {
+        $('.toolbar-pf-results h5').text(this.ctrl.totalCount + ' items');
+        if (this.ctrl.totalCount <= 1) {
+          $('.filter-pf.filter-fields input').attr('disabled', '');
+        } else {
+          $('.filter-pf.filter-fields input').removeAttr("disabled");
+        }
+      }
+    }, 50);
```

Comment:
Does this work with `0` instead of `50`? I think as long as you use `$timeout` it will run after the next digest loop, so the elements should be there.

What does it look like before the text replacements?